src/core/reducers/errors.js

```diff
@@ -41,12 +44,23 @@ function getMessagesFromError(error) {
     });
 
     if (apiMessages.length) {
-      messages = apiMessages;
+      data = { ...data, messages: apiMessages };
     } else {
       log.warn('API error response did not contain any messages', error);
     }
+
+    if (
+      error.response.data.detail &&
+      // TODO: switch to checking for a code after
+      // https://github.com/mozilla/addons-server/issues/5122
+      error.response.data.detail === 'Signature has expired.'
```

Comment:
Seems like, looking through the rest of the code, just an error code here that the component maps to a string, would be better and let us translate it.